tests/testcases/table_sample_long/expected.tsv

```diff
@@ -0,0 +1,17 @@
+SAMPLE	CHROM	FORMAT['AD'][SAMPLE][0] / FORMAT['DP'][SAMPLE] * INFO['MQ']
+SAMPLE_1	chr18	27.54098360655738
+SAMPLE_2	chr18	27.54098360655738
+SAMPLE_1	chr18	34.285714285714285
+SAMPLE_2	chr18	34.285714285714285
+SAMPLE_1	chr18	31.666666666666668
+SAMPLE_2	chr18	31.666666666666668

```

Comment:
I would prefer not to use fractional numbers in unit tests (formatting might change between Python versions)

Reply:
Right, they now get rounded to integers.